pintc/tests/cli.rs

```diff
@@ -121,8 +126,17 @@ fn default_output() {
     assert!(input_file.path().with_extension("json").exists());
     let _ = fs::remove_file(input_file.path().with_extension("json"));
 
+    let input_file_name = input_file.path().file_name().unwrap().to_str().unwrap();
+    let mut expected_output_file = tempfile::NamedTempFile::new().unwrap();
+    let expected_output = format!("    contract {input_file_name}       EB87FCE275A9AB10996D212F39221A56B90E01C37FA9D16EE04A3FE8E17DEED9
+         └── {input_file_name}::test BA6595C5C75346E6C82BED0CE770D0758ADD1712163FCE45E38E5E8EAC6AA153\n");
```

Comment:
No :(

I was unable to find a way to get that to work. 

When it was a string literal in `expect![` it could be updated, but would fail the next time the test was run. This is because the whitespace in front of `        contract` was being ignored even with a `raw string`.

If you know a way to preserve the whitespace in the string without the macro affecting it, then all problems would be solved :)

Reply:
Ok so here's my suggestion:
```rust
    let stdout = output.stdout.replace(
        input_file.path().file_stem().unwrap().to_str().unwrap(),
        "<filename>",
    );
    let stdout = stdout.trim();

    check(
        &stdout,
        expect_test::expect![[r#"
        contract <filename>       EB87FCE275A9AB10996D212F39221A56B90E01C37FA9D16EE04A3FE8E17DEED9
                 └── <filename>::test BA6595C5C75346E6C82BED0CE770D0758ADD1712163FCE45E38E5E8EAC6AA153"#]],
    );
```
This seem to work well. Updates with `expect-test` and does not care about trailing spaces.